opencivicdata/legislative/models/__init__.py

```diff
@@ -6,4 +6,4 @@
 from .vote import (VoteEvent, VoteCount, PersonVote, VoteSource)
 from .event import (Event, EventLocation, EventMedia, EventMediaLink, EventDocument, EventLink,
                     EventSource, EventParticipant, EventAgendaItem, EventRelatedEntity,
-                    EventAgendaMedia, EventAgendaMediaLink, EventDocumentLink)
+                    EventAgendaMedia, EventAgendaMediaLink, EventDocumentLink)
```

Comment:
Yeah, I'm not sure what happened exactly, as I would not have had any reason to change this file. Looks like the newline at the end of the file was deleted. I just added it back.